### PR TITLE
fix: [ANDROAPP-1153] category options restricted by orgUnits

### PR DESCRIPTION
**Pull request by mmmateos**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://jira.dhis2.org/browse/ANDROAPP-1153)

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/src/main/java/org/dhis2/Bindings/CategoryOptionExtensions.kt [hunk 1]
```diff
@@ -9,3 +10,11 @@ fun CategoryOption.inDateRange(date: Date?): Boolean {
...
     } ?: true
 }
+
+fun CategoryOption.inOrgUnit(orgUnitUid: String?): Boolean {
+    return orgUnitUid?.let {
+        organisationUnits()?.takeIf { it.isNotEmpty() }?.let {
+            orgUnitUid in UidsHelper.getUidsList(it)
+        } ?: true
+    } ?: true
+}
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/Bindings/CategoryOptionExtensions.kt:20]: Should this go to SDK logic @Balcan ?

Comment (Balcan) [reply app/src/main/java/org/dhis2/Bindings/CategoryOptionExtensions.kt:20]: Yes, this could be helpful in the SDK. We have to tell Victor

**Review by Balcan — APPROVED** [review]

File: app/src/main/java/org/dhis2/Bindings/CategoryOptionExtensions.kt (hunk 1 above)
Comment (JaimeToca) [reply app/src/main/java/org/dhis2/Bindings/CategoryOptionExtensions.kt:20]: Does it makes sense to move it to the logic package?

Comment (mmmateos) [reply app/src/main/java/org/dhis2/Bindings/CategoryOptionExtensions.kt:20]: Okey, we could move the whole class

**Pull request merged**